app/model/user.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const mongoose = require('mongoose');
+// const bcrypt = require('bcrypt');
```

Comment:
Unless this code is expected to be used later, commented code should be removed.